content/05kapitel.tex

```diff
@@ -2,8 +2,125 @@
 
 \chapter{Technische Umsetzung}
 
-\section{Mockups}
 
 \section{Architektur}
+Um unsere Produkt der öffentlichkeit zugänglich zu machen, heißt live Hosten und dabei auch einen angehenmen Workflow\footnote{Arbeitsablauf} für die Entwickler bereitzustellen, haben wir uns entschieden einen Server in der Cloud zu mieten um dies zu erfüllen.
```

Comment:
Der Satz Macht 0% Sinn.

Um unsere Plattform potenziellen Nutzern zugänglich zu machen und bestimmte Teile unseres Workflows (dt. Arbeitsablauf) zu Automatisieren, hosten wir unsere Architektur in einer Cloud Umgebung.

Reply:
fixed